components/board/numato/darwin_utils.go

```diff
@@ -0,0 +1,11 @@
+//go:build darwin
+
+package numato
+
+import "go.viam.com/utils/usb"
+
+func getSerialDevices() []usb.Description {
```

Comment:
This function needs some documentation: I can't tell what it does. Same comment on linux_utils.go.